src/vendor_upstream/stubs/EventListener.js

```diff
@@ -25,18 +25,18 @@ var EventListener = {
    * @param {function} callback Callback function.
    * @return {object} Object with a `remove` method.
    */
-  listen: function(target, eventType, callback) {
+  listen: function (target, eventType, callback) {
```

Comment:
nit: don't add this space between the function keyword and the parameters.  Here and throughout the files.